Src/PCompiler/CompilerCore/Backend/Rvm/AjFileGenerator.cs

```diff
@@ -70,47 +111,72 @@ public IEnumerable<CompiledFile> GenerateSources(Scope globalScope)
             Context.WriteLine(output, "    !within(javamoprt..*) &&");
             Context.WriteLine(output, "    !within(rvmonitorrt..*) &&");
             Context.WriteLine(output, "    !within(com.runtimeverification..*);");
-            Context.WriteLine(output, "}");
-            Context.WriteLine(output);
-        }
+            Context.WriteLine(output, "pointcut CommonPointCut() :");
+            Context.WriteLine(output, "    !within(com.runtimeverification.rvmonitor.java.rt.RVMObject+) &&");
+            Context.WriteLine(output, "    !adviceexecution() &&");
+            Context.WriteLine(output, "    notwithin();");
+            Context.WriteLine(output, "pointcut UnitTestCase() : ");
+            Context.WriteLine(output, "    execution(* junit.framework.TestCase+.test*(..)) ||");
+            Context.WriteLine(output, "    execution(* s3.junit.S3TestCase+.test*(..)) ||");
```

Comment:
Can we rename s3 to something else? We should not put anything specific to S3 in the P repo.

Reply:
I can remove this entirely.